packages/email-components/src/components/RatingItem/RatingItem.scss

```diff
@@ -15,6 +15,7 @@
     padding: theme.$padding-mobile;
     overflow: hidden;
     text-align: center;
+    text-decoration: none !important;
```

Comment:
I'm not performing manual tests. This property is no being overridden within style cascade by media queries (in many components does). That's why I've gotta ask if there is a reason to force the value with `!important` here?

Reply:
Yes, because some email clients add underlines to a link. This is the only way to overwrite it.